packages/sdk/src/handlers/relations.ts

```diff
@@ -1,15 +1,76 @@
 /**
  * Relations handler
  */
-
-import { ItemsHandler } from '../base/items';
+import { ManyItems, OneItem, PartialItem, QueryMany, QueryOne } from '../items';
 import { ITransport } from '../transport';
-import { RelationType, DefaultType } from '../types';
+import { RelationType, DefaultType, ID } from '../types';
 
 export type RelationItem<T = DefaultType> = RelationType & T;
+export class RelationsHandler<T = RelationItem> {
+	transport: ITransport;
 
-export class RelationsHandler<T = DefaultType> extends ItemsHandler<RelationItem<T>> {
 	constructor(transport: ITransport) {
-		super('directus_relations', transport);
+		this.transport = transport;
+	}
+
+	async readOne(collection: string, id: ID, query?: QueryOne<T>): Promise<OneItem<T>> {
+		const response = await this.transport.get(`/relations/${collection}/${id}`, {
+			params: query,
+		});
+		return response.data as T;
+	}
+
+	async readMany(collection: string, query?: QueryMany<T>): Promise<ManyItems<T>> {
+		const { data, meta } = await this.transport.get(`/relations/${collection}`, {
+			params: query,
+		});
+		return {
+			data,
+			meta,
+		};
+	}
+	async createOne(item: PartialItem<T>, query?: QueryOne<T>): Promise<OneItem<T>> {
+		return (
+			await this.transport.post<T>(`/relations/`, item, {
+				params: query,
+			})
+		).data;
+	}
+
+	async createMany(items: PartialItem<T>[], query?: QueryMany<T>): Promise<ManyItems<T>> {
+		return await this.transport.post<PartialItem<T>[]>(`/relations}`, items, {
```

Comment:
Typo in the endpoint

```suggestion
		return await this.transport.post<PartialItem<T>[]>(`/relations`, items, {
```

Reply:
also, relations doesn't have createMany